src/components/Footer.tsx

```diff
@@ -384,7 +396,7 @@ const Footer = () => {
                   {section.links.map((link, linkIdx) => (
                     <ListItem key={linkIdx}>
                       <FooterLink
-                        dir={isPageRightToLeft ? "auto" : "ltr"}
+                        dir={isPageRightToLeft ? "rtl" : "ltr"}
```

Comment:
`dir` can be `auto` https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir

Lets update the `Link` props to accept it `dir?: "ltr" | "rtl" | "auto"`